### ISSUE TITLE
Alt text translation is not viewable

### ISSUE THREAD

**Issue by Zoarial94** [pachli/pachli-android#1483]
**Describe the bug**
When translating a post, the translation is not available to view in the image, but it can be partially viewed by holding the image. 

**To Reproduce**
1. Find a post that needs its alt text translated
2. Click on the 'Translate' option
3. Wait to finish translating
4. Click the image to view the full alt text
5. Find the original untranslated alt text

**Expected behavior**
The translated alt text should be fully viewable

**Screenshots or video**
To large to upload. I will return with a better video. 

**Links**
https://framapiaf.org/@sima78/114392023899700393

**Versions**
Pachli 2.11.0

**Comment by Zoarial94** [comment]
https://github.com/user-attachments/assets/10248e1c-ab48-405c-88bb-f3319bf98ea7

**Comment by nikclayton** [comment]
Thanks for the report, and for linking to a post that showed the problem, that's very helpful.

The fix will be in the next Pachli release.
